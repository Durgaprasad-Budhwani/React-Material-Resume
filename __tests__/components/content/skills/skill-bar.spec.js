/**
 * Created by durgaprasad on 10/10/17.
 */
/* global expect,it */
import React from 'react';
import SKillBar from '../../../../src/components/content/skills/skill-bar';
import renderer from 'react-test-renderer';

it('renders skill bar', () => {
  const tree = renderer.create(
    <SKillBar
      name="name1"
      completed={60}
      size={10}
      color="primary"
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});