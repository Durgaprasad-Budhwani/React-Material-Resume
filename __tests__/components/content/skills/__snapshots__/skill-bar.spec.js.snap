// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders skill bar 1`] = `
<div
  className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 MuiGrid-grid-lg-6-81"
>
  <div
    className="MuiGrid-container-5 MuiGrid-item-6 MuiGrid-justify-xs-space-between-24 SkillBar-labelContainer-2"
  >
    <span
      className="SkillBar-label-1"
    >
      name1
    </span>
    <span
      className="SkillBar-label-1"
    >
      60%
    </span>
  </div>
  <div
    aria-valuenow={60}
    className="MuiLinearProgress-root-102 MuiLinearProgress-colorPrimary-103 MuiLinearProgress-determinate-105"
    role="progressbar"
  >
    <div
      className="MuiLinearProgress-bar-112 MuiLinearProgress-barColorPrimary-113 MuiLinearProgress-bar1Determinate-116"
      style={
        Object {
          "transform": "scaleX(0.6)",
        }
      }
    />
  </div>
</div>
`;
